vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
+- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+
+### Minimum Hardware Requirements
+
+The minimum hardware requirements depends on how you intend to use Entando 6. The more functionality you need such as a content management system (CMS), the more you will need.
+#### For Entando install only:
+
+- 6 vCPU or virtual central processing unit  (6000 millicores)
+- 6GB of RAM
+- 5GB of allocated Storage distributed in many PVCs
+
+Configure a single worker node with the requirements above or designate 3 worker nodes with the minimum of 2VPC and 4GB of RAM each.
+
+#### For Entando + Microservices:
+
+- 500 millicores per Microservice or 
+- 1000 millicores per Microservice with a DB
+
+The instance tested was generated with a JHipster Entando blueprint.
+
+#### For Entando with CMS: 
+
+To install Entando with CMS, the hardware requirements will depend on the complexity and number of pages and contents. The minimum requirements are:
+
+- 9vCPU for a single CMS instance. To scale up, you need 4vCPU per replica.
+- 8GB of RAM for a single CMS instance. To scale up,  you need 4GB of RAM per replica.
+- A minumum of 50GB of storage for all the static resources generated by the use of Entando CMS.
```

Comment:
instance tested -> tested instance
-> "complexity of content and number of pages"
"storage" not capitalized (was prior); second person; extra space

Reply:
1. agree
for 2, content is one of the components and used in that sense here, I think.
3. agree

Reply:
ok; i wasn't sure. should it be capitalized to make that clear and to read less awkwardly?